### DISCUSSION TITLE
[Bug] Trace data lose

### DISCUSSION THREAD

**Discussion by CodingMakeWordBetter** [apache/skywalking#12476] [Q&A] [answered]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar issues.

### Apache SkyWalking Component

OAP server (apache/skywalking)

### What happened

When I was monitoring my springboot project using Skywalking 9.7.0, I found that in some cases: calling an annotated @ Asynchronous method, some tasks are submitted to the thread pool in this method, and the thread pool will make HTTP requests, but I did not see these http request trace data in the SpringAsync view of the Skywalking UI

### What you expected to happen

I'm not sure if it's a problem with the OAP collection or if the agent hasn't reported it, but this situation can be easily reproduced.
I am not sure where the problem lies and cannot fix it. If I can provide assistance or prompts, I will try to fix it

### How to reproduce

I provide the following pseudocode：
`      
   @Resource(name = "messageTasksExecutor")
    private AsyncTaskExecutor messageTasksExecutor;
  
    @Async("scheduledTasksExecutor")
    @TransactionalEventListener(phase = TransactionPhase.AFTER_COMMIT)
    public void sendMessageListener(MessageBatchInfo messageBatchInfo) {
            for (WhatsAppMessage whatsAppMessage : whatsAppMessages) {
      
                messageTasksExecutor.execute(() -> {
                    // todo http request
                });
            }
    }`
I am certain that an HTTP request was made in messageTasksExecuter, but I did not see it in SpringAsynchronous in the Skywalking UI view

### Anything else

I used Alibaba's trace to detect my project before, and he used OpenTelemetery-JavaAgent. jar for data reporting without any loss.

### Are you willing to submit a pull request to fix on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
Every time, when you run into the async the scenario, you need to check whether there is a suitable plugin.

**Reply by CodingMakeWordBetter** [reply]
I understand that there is a plugin for bootstrap plugins called apm-jdk-threadpool-plugin-9.1.0.jar. Move it to the plugins folder and specify the jdkthreading. threading. class_prefixes=com prefix. But unlike OpenTelemetry as the default option? Does this feature have a significant impact on the performance of the system?

**Reply by wu-sheng** [reply, marked as answer]
I don't care how otel says. It's not our concern or conclusions.

Across thread scenario is always complex. We are confident there is no perfect solution. This plugin only fits for some cases.
